charts/catalog/templates/webhook-deployment.yaml

```diff
@@ -7,6 +7,10 @@ metadata:
     chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
     release: "{{ .Release.Name }}"
     heritage: "{{ .Release.Service }}"
+{{- if and (.Values.webhook.tls.caInjectFrom) (.Values.webhook.tls.secret) }}
+  annotations:
+    rollout.restart/auto: "true"
+{{- end }}
```

Comment:
maybe provide an independent var for adding annotations and then add the annotation itself via values file? 
This way, if more annotations have to be added, it will be easier

Reply:
sort of like 
~~~yaml
      {{ if .Values.webhook.annotations }}
      annotations:
{{ toYaml .Values.webhook.annotations | indent 8 }}
      {{- end }}
~~~

Reply:
Ok.

Reply:
But there's already `.Values.webhook.annotations` and it is used to add annotations to the pods (the more common scenario). So we would need to add a `.Values.webhook.deployment.annotations`.

I wonder how useful it is to preemptively make this more generic given that it's not common to add annotations to `Deployment`.

Reply:
https://github.com/hellofresh/service-catalog/pull/18/files#diff-877bbec07dd32b72e33f4f7f5e06b3d4cbdafb6549ac002f598fc12955e8c572R31

Reply:
ok then lets go with this for now